src/main/java/hudson/plugins/brakeman/BrakemanPublisher.java

```diff
@@ -101,7 +101,17 @@ public BuildResult publishReport(final Run<?, ?> build, final FilePath workspace
 			String output = brakemanOutput.readToString();
 
 			ParserResult project = new ParserResult(workspace);
-			this.scan(output, project);
+
+			AbstractBrakemanScanner scanner;
+
+			if(this.outputFile.endsWith(".json")) {
+				scanner = new BrakemanJSONScanner();
```

Comment:
So ignored warnings aren't currently triggers. That's a functionality for another iteration. This simply only evaluates the status of the build by the number of actual warnings.